### PR TITLE
Adds headpat gloves to arcade prizes

### PR DESCRIPTION
**Pull request by Ty-Omaha**
**What does this PR do:**
Adds the headpat gloves to the arcade machine prizes as requested in #12189 

:cl:
tweak: Headpat gloves added to arcade machine prizes
/ 🆑

### CODE REVIEW

**Comment by MrMagolor** [comment]
You know people will just immediately rush the arcade for this and then sit in front of bridge

**Comment by Ty-Omaha** [comment]
>You know people will just immediately rush the arcade for this and then sit in front of bridge

It's in a vending machine right now

**Comment by MrMagolor** [comment]
>It's in a vending machine right now

That explains a lot

**Comment by Fox-McCloud** [comment]
500 is an insane cost. The chainsaw costs 200.

100 to 150 is good for a gimmick item.

File: code/modules/arcade/prize_datums.dm [hunk 1]
```diff
@@ -318,6 +318,12 @@ var/global/datum/prizes/global_prizes = new
 	typepath = /obj/item/twohanded/toy/chainsaw
 	cost = 200
 
+/datum/prize_item/headpat
+	name = "Gloves of Headpats"
+	desc = "Gloves that let you heatpat fast."
```
Comment (Fox-McCloud) [inline code/modules/arcade/prize_datums.dm]: "Gloves that fill you with an irresistable urge to give headpats."

Most of the arcade prizes have something witty about them

**Comment by Ty-Omaha** [comment]
@Fox-McCloud done

**Pull request merged**